test/Experimental/ExteriorAlgebra-test.jl

```diff
@@ -4,28 +4,31 @@
 
 ######## CONSTRUCTOR TESTs
 @test_throws  ArgumentError  exterior_algebra(QQ, 0);
-@test_broken  exterior_algebra(QQ, 1);  # --> special case (is commutative)
```

Comment:
I don't know what the plan with this PR is (whether it will be merged soon or still needs work or whatever). But if the merge is not imminent, could you please put this fix into a separate PR, so that we get rid of the additional CI failure?

Thanks!